### PR TITLE
[PIL-847] Add new liquidity pools

### PR DESCRIPTION
**Pull request by graszka22**
* Add 20 new Uniswap liquidity pools (highest USD volume) - only on Prod env
* There is a script responsible for fetching these pools where you copy-paste the output to update source
* Only supported pools will be displayed (pool address is supported, as well as each of the pool tokens)
* These pools are not to supporting token staking, but the existing ETH/PLR, DAI/PLR should continue to support it
* New pools use generic uniswap icon (agreed with Dmitry)
* Various tweaks: price display, hidden axis labels on charts

### CODE REVIEW

File: src/utils/assets.js [hunk 1]
```diff
@@ -254,6 +254,14 @@ export const addressesInclude = (addresses: string[], addressToFind: ?string): b
   return addresses.some(item => isCaseInsensitiveMatch(item, addressToFind));
 };
 
+export const isSupportedAssetAddress = (supportedAssets: Asset[], addressToCheck: ?string) => {
+  return supportedAssets.some((asset: Asset) => addressesEqual(asset.address, addressToCheck));
+};
+
+export const isSupportedAssetSymbol = (supportedAssets: Asset[], symbolToCheck: ?string) => {
+  return supportedAssets.some((asset: Asset) => asset.symbol === symbolToCheck);
+};
```
Comment (jdnmr) [inline src/utils/assets.js:263]: I know we lack them in most places, but lately we've been adding return types too

**Review by jdnmr — APPROVED** [review]
I'm out of ETH to test, but codewise looks good! See a few comments, especially the one about theGraph calls.

File: scripts/fetchLiquidityPools.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        iconUrl: 'asset/images/tokens/icons/uniColor.png',
+      },`,
+      );
+    });
+  } catch (error) {
+    console.error('🔴 Script Error 🔴', error);
+  }
+};
+
+main();
```
[the remark this answers is not in the document]
Comment (mdjastrzebski) [reply scripts/fetchLiquidityPools.js:78]: atm available LPs are hardcoded, and this is a script that was used to generate current list. It's not part of the app but rather a standalone script that generates LP data to be copy-pasted to app source.

File: src/constants/liquidityPoolsConstants.js [hunk 3]
```diff
@@ -33,30 +35,33 @@ export const LIQUIDITY_POOLS_STAKE_TRANSACTION = 'LIQUIDITY_POOLS_STAKE_TRANSACT
...
+export const UNIPOOL_LIQUIDITY_POOLS = (): UnipoolLiquidityPool[] => {
+  if (getEnv().ENVIRONMENT === STAGING) {
+    return [
+      {
+        name: 'Uniswap v2 ETH-PLR',
+        type: LIQUIDITY_POOL_TYPES.UNIPOOL,
+        symbol: 'UNI-V2',
+        tokensProportions: [
+          { symbol: 'ETH', proportion: 0.5, progressBarColor: '#497391' },
+          { symbol: 'PLR', proportion: 0.5, progressBarColor: '#00ff24' },
```
[the remark this answers is not in the document]
Comment (mdjastrzebski) [reply src/constants/liquidityPoolsConstants.js:47]: Likewise, this is not part of the app, but standalone script.

File: src/utils/assets.js (hunk 1 above)
Comment (mdjastrzebski) [reply src/utils/assets.js]: Added return types.

File: scripts/fetchLiquidityPools.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mdjastrzebski) [reply scripts/fetchLiquidityPools.js:78]: It could be useful if we want to modify the list for whatever reason.

**Pull request merged**